templates/slim/section.html.slim

```diff
@@ -1,6 +1,9 @@
 / hide slides on %conceal, %notitle and named "!"
 - titleless = (title = self.title) == '!'
 - hide_title = (titleless || (option? :notitle) || (option? :conceal))
+
+/ slide options processing
+- data_background = (attr? 'background') ? (image_uri(attr 'background')) : nil
```

Comment:
What about non image backgrounds? For reveal.js we have an option which doesn't seem supported
```html
<section data-background="#ff0000">
    <h2>All CSS color formats are supported, like rgba() or hsl().</h2>
</section>
```

```asciidoc
[background="#eee"]
== What are we doing?
```

is currently rendered into
```html
<section data-background="images/#eee">
   <h2>What are we doing?</h2>
</section>
```